db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -1069,196 +1069,52 @@ class NonBatchedOpsStressTest : public StressTest {
 
     int error_count = 0;
 
-    auto handle_result = [](ThreadState* _thread, const Status& s,
```

Comment:
Small review tip: it might be easier to review this with whitespace changes hidden